Model/Order/CompleteOrderPool.php

```diff
@@ -15,6 +15,7 @@
  */
 class CompleteOrderPool implements CompleteOrderInterface
 {
+    public const API_TYPE_SIMPLE = 'simple';
```

Comment:
^[SUGGESTION] this variable already exists so instead of duplicating it if we could re-use it like here, but use the default variable. 

https://github.com/bold-commerce/adobe-commerce-bold-checkout/blob/1ff38325058a491f18d5cc8a3f3c08d32ec7552a/Model/Order/InitOrderFromQuote.php#L178

Reply:
Updated!